### PR TITLE
manifest.json Erstellt 

### PR DESCRIPTION
**Pull request by NextPlatziReal**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by NextPlatziReal** [comment]
.

**Review by maaaathis — CHANGES_REQUESTED** [review]
- directory name has to be lowercase

- icon.png is not given, but it is required.

- icon@2x.png is not given, but it is required.

File: minecraft_servers/PlatziLegends/background.png
Comment (DoJapHD) [inline minecraft_servers/PlatziLegends/background.png:1]: Compress image

File: minecraft_servers/PlatziLegends/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/PlatziLegends/background@2x.png:1]: Compress image

**Review by DoJapHD — COMMENTED** [review]
Change folder name to lowercase

**Comment by DoJapHD** [comment]
Our check is faulty, ignore the error

**Comment by DoJapHD** [comment]
Again, our script is broken right now.
Please wait.

**Comment by DoJapHD** [comment]
Inactive PR

**Pull request closed without merging**